selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ToastLoader.java

```diff
@@ -33,37 +33,37 @@
 import org.openqa.selenium.support.ui.ExpectedCondition;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
-/**
- * //
- *
- * @author Musienko Maxim
- */
+/** @author Musienko Maxim */
 @Singleton
 public class ToastLoader {
   private final SeleniumWebDriver seleniumWebDriver;
+  private final SeleniumWebDriverHelper seleniumWebDriverHelper;
+  private final Loader loader;
 
   @Inject
-  public ToastLoader(SeleniumWebDriver seleniumWebDriver) {
+  public ToastLoader(
+      SeleniumWebDriver seleniumWebDriver,
+      SeleniumWebDriverHelper seleniumWebDriverHelper,
+      Loader loader) {
     this.seleniumWebDriver = seleniumWebDriver;
+    this.seleniumWebDriverHelper = seleniumWebDriverHelper;
+    this.loader = loader;
     PageFactory.initElements(seleniumWebDriver, this);
   }
 
   private interface Locators {
-    String MAIN_FORM = "gwt-debug-popupLoader";
-    String START_BUTTON = "//button[text()='Start']";
-    String MAINFORM_WITH_TEXT_CONTAINER =
+    String MAIN_FORM_ID = "gwt-debug-popupLoader";
+    String TOAST_LOADER_BUTTON_XPATH = "//div[@id='gwt-debug-popupLoader']//button[text()='%s']";
```

Comment:
Let's name it `TOAST_LOADER_BUTTON_XPATH_PATTERN` to make it obvious that it requires modification before usage.